src/riak_core_app.erl

```diff
@@ -145,3 +146,12 @@ register_capabilities() ->
       end,
       Capabilities),
     ok.
+
+%% TODO: This belongs in riak_kv - an issue will be created to move it, but time constraints.
+start_eleveldb_info_service() ->
+    Registration = {eleveldb, set_metadata_pid, []},
+    Shutdown = undefined, %%{eleveldb, metadata_service_shutdown, []}
+    InfoSource = {riak_core_bucket, get_bucket, []},
+    ResultsHandler = {eleveldb_metadata, handle_metadata_response, []},
+    {ok, _Pid} = riak_core_info_service:start_service(Registration, Shutdown, InfoSource, ResultsHandler),
+    ok.
```

Comment:
Here's a nickel, kid. Get yourself a real editor.

(With apologies to Scott Adams. http://dilbert.com/strip/1995-06-24)

(No end of line marker on the last line in the file.)